### ISSUE TITLE
Setting specific font makes CotEditor crash

### ISSUE THREAD

**Issue by takasek** [coteditor/CotEditor#1320]
### Description

See the crash log placed [here](https://gist.github.com/takasek/c7202268149e43d42f78bd9f683f0d60) because it was too long for the comment and leaded to the error that `There was an error creating your issue: body is too long (maximum is 65536 characters).`

### To Reproduce

1. just try to launch.

It may be an existing fault. I first met with the fault on Apr 12th.
I'm not sure what is the cause, but before the crash happened, I...

1. edited fonts in `~/Library/Fonts/` by fontforge
2. tried to change font on CotEditor's setting view

Right after clicking something about font setting (unclear, sorry...) the app crashed and never launched again.

### CotEditor version

4.2.3

### macOS version

11.6.6

**Comment by 1024jp** [comment]
Hi takasek, thank you for the report.

Because I could not reproduce your issue (even on macOS 11), I'd like to collect some more information.
Could you tell me the result of the following procedures?

1. The return value when you run the following lines in Terminal:
(These commands just check the current user preferences of the editor font for CotEditor.)
```sh
defaults read com.coteditor.CotEditor fontName
defaults read com.coteditor.CotEditor fontSize 
```

2. What happens when running the following lines in Terminal and then relaunching CotEditor?
(The lines reset the editor font to the default)
```sh
defaults delete com.coteditor.CotEditor fontName
defaults delete com.coteditor.CotEditor fontSize 
```

3. What happens when launching CotEditor while pressing the Shift key?
(Pressing the Shift key while launching surpresses reopening documents in the last sesion.)

**Comment by takasek** [comment]
Hi, thank you for the information.

> 1. The return value when you run the following lines in Terminal:
(These commands just check the current user preferences of the editor font for CotEditor.)

```
% defaults read com.coteditor.CotEditor fontName
Raanana
% defaults read com.coteditor.CotEditor fontSize
13
```

> 2. What happens when running the following lines in Terminal and then relaunching CotEditor?
(The lines reset the editor font to the default)

They worked.  After running two commands, CotEditor launched finely.  So I couldn't try procedures 3 .
Now a setting window shows the font `Helvetica 13` .

For an experimental trial I reselected `Raanana` again and it caused of the crash!
https://user-images.githubusercontent.com/1290052/172392389-c4601e5a-8427-405d-8259-a712d9ed0998.mov

But after that I could relaunch the app.  the font set to Raanana and I don't know why.
<img width="567" alt="image" src="https://user-images.githubusercontent.com/1290052/172393003-bc79329f-ba07-48ec-bbd6-30c12cbef424.png">

**Comment by takasek** [comment]
Raanana is a preinstalled font, which exists in `/System/Library/Fonts/Supplemental` .
I tried the same way on another machine (macOS Monterey 12.3.1) and I reproduced the crash.  

1. show `表示>フォント>選択…`
2. select `Raanana` font

If you couldn't reproduce the crash on your environment, I'll send my fonts.

Moreover, both of Raanana fonts installed on each machine differed on filesizes. It might be because of OS version.  However, at least on my environment they both made crash.
